### DISCUSSION TITLE
Questions: 1) How can I use NocoDB with SQLite? 2) How do I enter this view on desktop?

### DISCUSSION THREAD

**Discussion by WantToLearnJapanese** [nocodb/nocodb#605] [Discord Threads]
1) When I try to edit the column type (, all the settings are disabled and say I am using SQLite DB.
  Is NocoDB supposed not to fully work with SQLite? (I proper stick to SQLite since it would be used by only me)

2) On mobile, If I swipe to the left, I can enter this view, where I can edit types even on SQLite.
But how do I enter this view on Desktop?
![Screenshot_20211003-174806_Kiwi Browser.jpg](https://user-images.githubusercontent.com/88884927/135746607-f54ca6ed-f2ed-4939-9ac8-ff76fd0d324f.jpg)

**Comment by o1lab** [comment]
> On mobile, If I swipe to the left, I can enter this view, where I can edit types even on SQLite.
But how do I enter this view on Desktop?

Oh never knew that is possible. This view is for legacy reason - we will remove it. As of now its hidden from the user.


NocoDB works with SQLite well. 

However, in SQLite - column once created cannot be edited for its type or properties. Hence the options are disabled.

If you want to really stick to SQLite. Make the changes to SQLite schema from another tool / code you are comfortable with and connect NocoDB to it.

**Reply by WantToLearnJapanese** [reply]
Never knew such limitations on SQLite itself.

**Reply by o1lab** [reply]
https://stackoverflow.com/questions/21199398/sqlite-alter-a-tables-column-type/21199519#:~:text=SQLite%20does%20not%20fully%20support,to%20rename%20the%20new%20one.

**Comment by o1lab** [comment]
Moving this to discussion
